doc/statuspage.md

```diff
@@ -0,0 +1,57 @@
+# StatusPage
+
+This document relates to the [StatusPage](https://www.statuspage.io/) for Node.js, which can be
+found at [status.nodejs.org](https://status.nodejs.org).
+
+## Access
+
+The following people have access to the StatusPage control panel. New accounts can be added via the
+[User Management](https://manage.statuspage.io/organizations/m3074zj4qnr9/team) page.
+
+Access to StatusPage is unrestricted, so anyone in this list can make changes to the styling of
+the page, change what components are listed, see who is subscribed to notifications (PII: emails,
+phone numbers) and manage incidents on the page.
+
+The only additional "permission" is that one email address is considered the owner of the
+status page. This account cannot be removed by other members and acts as the official owner of the
+page for communication with the StatusPage folks if needed.
+
+| GitHub                                         | Email (for StatusPage access)  | Role   |
+|------------------------------------------------|--------------------------------|--------|
+| [@MattIPv4](https://github.com/MattIPv4)       | nodejs-status@mattcowley.co.uk | Owner  |
+| [@brianwarner](https://github.com/brianwarner) | operations@openjsf.org         | Member |
+
+## Twitter
+
+There is also a Twitter account, [@NodejsStatus](https://twitter.com/NodejsStatus), for the
+status page.
+
+This account is run by [@MattIPv4](https://github.com/MattIPv4) via nodejs-status@mattcowley.co.uk,
```

Comment:
Is there a reason this is not going to be a repo under the node.js org?

Reply:
Hm?

Reply:
My question is if there is a reason that the twitter account is run by you personally versus being something managed by the project with you being one of the people who can post to it.

Reply:
But I guess that is covered below.

Reply:
Yeah, I made it at the same time as setting up the status page itself. Going forward, no one should actually need to ever post manually to it, as status page automation will do that, so haning it over the ops@jsf makes sense once we're ready to go.